ReactQt/tests/reacttestcase.h

```diff
@@ -10,23 +10,36 @@
 class ReactView;
 class ReactBridge;
 
+#define INIT_TEST_CASE_DEFAULT(BaseClass)                                                                              \
+    virtual void initTestCase() override {                                                                             \
+        BaseClass::initTestCase();                                                                                     \
+    }
+
+#define CLEANUP_TEST_CASE_DEFAULT(BaseClass)                                                                           \
+    virtual void cleanupTestCase() override {                                                                          \
+        BaseClass::cleanupTestCase();                                                                                  \
+    }
```

Comment:
@vkjr have added macros to define default implementation of init and cleanup unit tests' methods